cinema/views.py

```diff
@@ -127,6 +125,35 @@ def upload_image(self, request, pk=None):
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                name="title",
+                type=OpenApiTypes.STR,
+                description="Filter movie by title",
+            ),
+            OpenApiParameter(
+                name="genres",
+                type={
+                    "type": "array",
+                    "items": {"type": "number"}
+                },
+                description="Filter movie by genres"
+            ),
+            OpenApiParameter(
+                name="actors",
+                type={
+                    "type": "array",
+                    "items": {"type": "number"}
+                },
+                description="Filter movie by actors"
+            )
+        ]
+    )
```

Comment:
worth to mention that these parameters are not required? (OpenApiParameter has a param for that purpose)